client/src/parser.js

```diff
@@ -0,0 +1,99 @@
+const { parse } = require('path');
+
+/**
+ * @description Parses an JSON created from the .bpmn xml of the model to an executable .robot file
+ * @returns {string} Code that has to be put in .robot file
+ */
+
+// this should be refactored to a dictionary
```

Comment:
If this is supposed to happen, why did it not happen yet?

Reply:
not enough time